subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/DetachedConfigurationsIntegrationTest.groovy

```diff
@@ -65,4 +66,99 @@ class DetachedConfigurationsIntegrationTest extends AbstractIntegrationSpec {
         expect:
         run "checkDependencies"
     }
+
+    def "detached configurations may have project dependencies"() {
```

Comment:
```suggestion
    def "detached configurations may have dependencies on other projects"() {
```

Clarified to differentiate this test from the ones below which depend on the current project